docs/book/how-to/manage-the-zenml-server/best-practices-upgrading-zenml.md

```diff
@@ -0,0 +1,54 @@
+# Best practices for upgrading ZenML
+
+While upgrading ZenML is generally a smooth process, there are some best practices that you should follow to ensure a successful upgrade. Based on experiences shared by ZenML users, here are some key strategies and considerations.
+
+## Upgrading your server
+
+Follow the tips below while upgrading your server to mitigate data losses, downtime and other issues.
+
+### Data Backups
+
+- **Database Backup**: Before upgrading, create a backup of your MySQL database. This allows you to rollback if necessary.
```

Comment:
maybe give a bash command for this?

Reply:
tbh as its so individual for every type of MySQL deployment, I would rather disagree and say don't give a bash command